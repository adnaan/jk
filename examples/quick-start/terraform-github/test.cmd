jk generate --stdout github.js
