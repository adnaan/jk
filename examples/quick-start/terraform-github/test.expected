"provider" "github" {
  "organization" = "myorg"
}

"github_membership" "myorg_alice84" {
  "username" = "alice84"

  "role" = "admin"
}

"github_membership" "myorg_bob93" {
  "username" = "bob93"

  "role" = "member"
}
